docs/src/manual/interface.md

```diff
@@ -13,6 +13,18 @@ You can read the documentation of [TermInterface.jl](https://github.com/JuliaSym
 
 ## SymbolicUtils.jl only methods
 
-`promote_symtype(f, arg_symtypes...)`
+### `symtype(x)`
+
+Returns the symbolic type of `x`. By default this is just `typeof(x)`.
```

Comment:
```suggestion
Returns the 
[numeric type](https://docs.julialang.org/en/v1/base/numbers/#Standard-Numeric-Types) 
of `x`. By default this is just `typeof(x)`.
```

How about numeric type? Symbolic type is a bit unclear.

Reply:
Question is, do we really need this default?

Reply:
I think so. While Symbolics.jl primarily focuses on [`BasicSymbolic`](https://github.com/JuliaSymbolics/SymbolicUtils.jl/blob/43797eb43b33b8d6b0374eac8cd6f87f94bcf79d/src/types.jl#L26) objects, there are instances where it can be used with other types. 
For example, https://github.com/JuliaSymbolics/Symbolics.jl/issues/1157#issue-2334289624